3_visualize/src/plot_functions.R

```diff
@@ -0,0 +1,74 @@
+
+#function to create plot of data and save as .png file. Also saves the model diagnostics
+#text file to the same output folder - wasn't sure where to save that in our directory structure.
+
+
+plot_data<-function(eval_data){
+  
+  # Create a plot
+  png(file = file.path('./3_visualize/out/figure_1.png'), width = 8, height = 10, res = 200, units = 'in')
```

Comment:
For plot functions, it's good to add arguments for commonly edited plot features. Input data, output save path, plot axes, plot dimensions, etc.